challenges/dailyRevenue/dailyRevenue.js

```diff
@@ -0,0 +1,21 @@
+function dailyRevenue(transactions) {
+  let results = {};
+
+  transactions.map(purchase => {
```

Comment:
Use a forEach or reduce instead of map - you aren't returning the new array here.